receiver/hostmetricsreceiver/internal/scraper/memoryscraper/metadata.yaml

```diff
@@ -6,7 +6,7 @@ attributes:
   state:
     description: Breakdown of memory usage by type.
     type: string
-    enum: [buffered, cached, inactive, free, slab_reclaimable, slab_unreclaimable, used]
+    enum: [buffered, cached, inactive, free, slab_reclaimable, slab_unreclaimable, used, available]
```

Comment:
Can you please expand the list in multiple lines and add comments for the states that are disabled by default

Reply:
good idea  👍

Reply:
done